src/utils/datastructs.jl

```diff
@@ -3,22 +3,31 @@ abstract type OptData <: InputData end
 
 """
      OptModelCEP
--model::JuMP.Model
--info::Array{String}
--set::Dict{String,Array}
+The essential elements of a CapacityExpansionProblem are organized as a `OptModelCEP` struct:
+- `model`::JuMP.Model - contains the actual JuMP-Model
+- `info`::Array{String} - contains the information about the model setup in form of multiple lines of equations
+- `set`::Dict{String,Array} - contains the sets used by the model
 """
 struct OptModelCEP
   model::JuMP.Model
   info::Array{String}
-  set::Dict{String,Array}
+  set::Dict{String,Dict{String,Array}}
```

Comment:
What is the reasoning behind this change?

Reply:
check for example